### DISCUSSION TITLE
Seplos BMS v3 support status

### DISCUSSION THREAD

**Discussion by msevestre31** [Louisvdw/dbus-serialbattery#1143] [Q&A] [answered]
Hello

I found that seplos BMS v3 was worked out in the list of BMS.

Is this up to date and still work in progress ? What is the current status ?

Thanks
Matt

**Comment by mr-manuel** [comment]
Check the documentation, there you find everything you need.

**Reply by msevestre31** [reply]
Hi @mr-manuel.

I indeed took a look, already, but found misleading data, facing two different type of information:
- one stating _« Working on the Seplos v3 BMS which uses different (modbus) protocol than the earlier Seplos models. »_ ("[BMS supported](https://mr-manuel.github.io/venus-os_dbus-serialbattery_docs/general/supported-bms
)" page), thus "work in progress but not working" (with no status nor date) - 
- one proving a [list of features](https://mr-manuel.github.io/venus-os_dbus-serialbattery_docs/general/features#bms-feature-comparison) that leads me to understand the "BMS supported" page information is outdated.

Which one is correct ?

Thanks

**Reply by mr-manuel** [reply]
"Working on the Seplos v3 BMS which..." --> "The driver works on the Seplos v3 BMS which..."

**Reply by mr-manuel** [reply, marked as answer]
Or also "Works for the Seplos v3 BSM which..."

**Reply by msevestre31** [reply]
OK, wording is a bit misleading.

"Working on ... = work in progress.

Definitely "**Works** on the Seplos v3 BMS..." would clarifies things. Thanks !

Matt

**Reply by mr-manuel** [reply]
I already updated the wording ;-)
